packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx

```diff
@@ -153,49 +153,49 @@ export const RecentThreadsHeader = ({
 
   return (
     <div className="RecentThreadsHeader">
-      {isUndefined(topic) && (
-        <>
-          <div className="header-row">
-            <CWText type="h3" fontWeight="semiBold" className="header-text">
-              All Discussions
-            </CWText>
-            <div className="count-and-button">
-              <CWText
-                type="caption"
-                fontWeight="medium"
-                className="thread-count-text"
-              >
-                {totalThreadCount} Threads
-              </CWText>
-              {windowIsExtraSmall ? (
-                <CWIconButton
-                  iconName="plusCircle"
-                  iconButtonTheme="black"
-                  onClick={() => {
-                    navigate('/new/discussion');
-                  }}
-                  disabled={!app.user.activeAccount}
-                />
-              ) : (
-                <CWButton
-                  buttonType="mini-black"
-                  label="Create Thread"
-                  iconLeft="plus"
-                  onClick={() => {
-                    navigate('/new/discussion');
-                  }}
-                  disabled={!app.user.activeAccount}
-                />
-              )}
-            </div>
-          </div>
-          <CWText className="subheader-text">
-            This section is for the community to discuss how to manage the
-            community treasury and spending on contributor grants, community
-            initiatives, liquidity mining and other programs.
+      <div className="header-row">
+        <CWText type="h3" fontWeight="semiBold" className="header-text">
+          {isUndefined(topic) ? 'All Discussions' : topic}
+        </CWText>
+        <div className="count-and-button">
+          <CWText
+            type="caption"
+            fontWeight="medium"
+            className="thread-count-text"
+          >
+            {totalThreadCount} Threads
           </CWText>
-        </>
+          {windowIsExtraSmall ? (
+            <CWIconButton
+              iconName="plusCircle"
+              iconButtonTheme="black"
+              onClick={() => {
+                navigate('/new/discussion');
+              }}
+              disabled={!app.user.activeAccount}
+            />
+          ) : (
+            <CWButton
+              buttonType="mini-black"
+              label="Create Thread"
+              iconLeft="plus"
+              onClick={() => {
+                navigate('/new/discussion');
+              }}
+              disabled={!app.user.activeAccount}
+            />
+          )}
+        </div>
+      </div>
+
+      {isUndefined(topic) && (
+        <CWText className="subheader-text">
+          This section is for the community to discuss how to manage the
```

Comment:
When the topic is not undefined, we should show the Topic's Description here too. 

When "All Discussions" (topic undefined), this description here should be changed. I will work on some UX copy for this imminently so we can fix this within the PR.

Reply:
[Done](https://github.com/hicommonwealth/commonwealth/pull/4157/commits/19f97f3375aa470d1b95def2effd6b27b4dbd0f6)

Reply:
Actually, let's remove default description text altogether for the "All Discussions" page. Only Topics with Descriptions should render a description.

Reply:
Otherwise, this PR looks ready to go 👍

Reply:
cc @mzparacha